### DISCUSSION TITLE
.

### DISCUSSION THREAD

**Discussion by sinahnyazdi** [WeAreMahsaAmini/Core#114] [Ideas]

**Comment by AnonymousMaximus2** [comment]
The idea is great but 

you need infrastructure for this idea, 

and I explained this infrastructure here:

https://github.com/WeAreMahsaAmini/Core/discussions/115

**Comment by redflake** [comment]
I would follow also the same idea!

**Comment by babooloo** [comment]
Why can't people just use bitcoin (or other well established cryptos)? Why do we need a new one?

**Reply by arashi-dev** [reply]
agree. 
maintaining a new token/coin takes so much time and resources. we need expert developers and not for only 1 month. but maintaining it for years. 
people don't like to buy a token that may have bugs and no one would buy that from them in the future. so users should never ever decrease.

if we are strong enough to invite people to use an Iranian token, then why we just don't teach them how to use bitcoin or alts and how it can help them?
we can make some contents of the pros. and the easiest tools for using it (wallet, buying, selling, etc.)

**Reply by babooloo** [reply]
I like the idea of creating content to teach people how to use bitcoin (or others). 
I haven't looked into cryptos that much since 2018. Are they now more user friendly? Does any type of ecosystem (e.g. exchanges etc) exist around them in Iran currently?

**Reply by arashi-dev** [reply]
I am not active too...
Cryptos are more user-friendly (a lot). but I have no idea about the ecosystem. There are a lot of limitations due to sanctions but I think there are also solutions (I guess exchange is the hardest part)
